localsettings/services-parsoid.php

```diff
@@ -4,9 +4,16 @@
 $wgEnableRestAPI = true;
 if ( file_exists( 'parsoid/extension.json' ) ) {
 	// Ensure we load the right Parsoid (not via Composer)
-	AutoLoader::$psr4Namespaces += [
-		'Wikimedia\\Parsoid\\' => 'parsoid/src',
-	];
+	if ( method_exists( AutoLoader::class, 'registerNamespaces' ) ) {
```

Comment:
This has existed since https://gerrit.wikimedia.org/r/c/mediawiki/core/+/773229 in REL1_39; are you trying to support REL1_38/REL1_35 still here? Might be worth a comment explaining when this can be dropped.